### PR TITLE
Fix build and coverage

### PR DESCRIPTION
**Pull request by mseri**
Signed-off-by: Marcello Seri <marcello.seri@citrix.com>

### CODE REVIEW

File: .coverage.sh [hunk 1]
```diff
@@ -10,11 +10,10 @@ if [ -z "$KEEP" ]; then trap "popd; rm -rf $COVERAGE_DIR" EXIT; fi
 
 $(which cp) -r ../* .
 
-opam pin add bisect_ppx --dev-repo -y
+opam pin add bisect_ppx 1.3.0 -y
 opam install ocveralls -y
 
-export COVERAGE=1
-jbuilder runtest
```
Comment (gaborigloi) [inline .coverage.sh]: Was this removed deliberately?

Comment (mseri) [reply .coverage.sh]: No, runtest should have stayed

**Comment by gaborigloi** [comment]
Looks like it's still adding some flags that `bisect_ppx` doesn't understand:
```
(cd _build/default && ./.ppx/bisect_ppx+cstruct.ppx+ppx_sexp_conv/ppx.exe -conditional -no-check --dump-ast --cookie 'library-name="nbd"' -o lib/channel.pp.ml --impl lib/channel.ml)
./.ppx/bisect_ppx+cstruct.ppx+ppx_sexp_conv/ppx.exe: unknown option '-no-check'.
ppx.exe [options] [<files>]
  --as-ppx                  Act as a -ppx rewriter
  --as-pp                   Shorthand for: --dump-ast --embed-errors
  --dump-ast                Output a binary AST instead of source code
  -o FILE                   Output to this file instead of the standard output
  --intf FILE               Treat FILE as a .mli file
  --impl FILE               Treat FILE as a .ml file
  --embed-errors            Embed error reported by rewriters into the AST
  --cookie NAME=EXPR        Set the cookie NAME to EXPR
  -exclude <pattern>        Exclude functions matching pattern
  -exclude-file <filename>  Exclude functions listed in given file
  -mode {safe|fast|faster}
                            Ignored
  -conditional              Do not instrument unless environment variable BISECT_ENABLE is YES
  -runtime <module          name>  Set runtime module name; used for testing
  -simple-cases             Do not generate separate points for clauses of or-patterns
  -inexhaustive-matching    Generate inexhaustive match expressions in cases; used for testing
  -help                     Display this list of options
  --help                    Display this list of options
```

**Comment by mseri** [comment]
Yes, it is because we are in a limbo using old libraries when we could move to the latest and avoid the hack. I forced the use of the type_conv in xs-opam so that it works with the hack. We can move to the new libraries more easily after we finish the jbuilder port and cleanup the opam files

**Pull request merged**